### ISSUE TITLE
project tslint.json rules are not working as expected

### ISSUE THREAD

**Issue by nevercast** [google/ts-style#174]
I've a tslint.json file that I expected would replace the gts one (I would, in practice, use the extends functionality to only override gts but have elected to not do that for the sake of minimizing the surface of the problem).

`tslint.json` is in the same directory that I am running gts from, I am running gts via `npx` with the `--no-install` flag so prevent gts from being installed. I've manually installed gts as a dev-dependency for my project.

My example `tslint.json` file is as follows:
```
{
  "rules": {
    // Disable no-default-export for now. Until I can rewrite the routes in to classes like I desired.
    "no-default-export": false
  }
}
```

It is my understanding that `tslint.json` supports comments (JSON with Comments format). This is my `tsconfig.json`

```
{
	"compilerOptions": {
		"allowSyntheticDefaultImports": true,
		"allowUnreachableCode": false,
		"allowUnusedLabels": false,
		"declaration": true,
		"esModuleInterop": true,
		"forceConsistentCasingInFileNames": true,
		"lib": [
			"es2016"
		],
		"module": "commonjs",
		"noEmitOnError": true,
		"noFallthroughCasesInSwitch": true,
		"noImplicitReturns": true,
		"pretty": true,
		"sourceMap": true,
		"strict": true,
		"target": "es2016",
		"moduleResolution": "node",
		"allowJs": true,
		"outDir": ".tsautobuild",
		"rootDir": "."
	},
	"typeAcquisition": {
		"enable": true
	},
	"compileOnSave": false,
	"include": [
		"src/*.ts",
		"src/**/*.ts",
		"test/*.ts",
		"test/**/*.ts"
	],
	"exclude": [
		"node_modules"
	]
}
```

This is mostly copy-paste from gts's tsconfig, as I did not like hiding parameters that other developers may not expect. 

From `package.json`, this is the relevant parts.

```
  "scripts": {
    "check": "npx -q --no-install gts check",
    "clean": "npx -q --no-install gts clean",
    "fix": "npx -q --no-install gts fix",
    "posttest": "npm run check"
  },
  "devDependencies": {
    "gts": "^0.6.0",
    "tslint": "^5.10.0",
    "typescript": "^2.9.1",
  },
```

When I run `npm run check`, and I see that `npx -q --no-install gts check` is executed, I expect the result to not include errors involving default exports. However I see results like this:

```
Use of default exports is forbidden (no-default-export)
  2 | import { byId } from '../../db/user';
  3 |
> 4 | export default async function(req, res) {
    |       ^
  5 |     const JWT = await openJWTFromHeader(req.headers.authorization);
  6 |     const subject = JWT.sub;
  7 |     const user = await (await byId(subject)).populate('role').populate('organization').execPopulate();
```

Until such time that we have a Sprint to rewrite our express routes to not use default exports, I would like gts to not flag this as a violation of style. Please help me remedy this, be the issue a fault in my configuration, or a fault in `google/ts-style`.

Thanks

**Comment by nevercast** [comment]
After looking at the built output of the installed `gts` module, it seems that the published version I have may not match the current source of this repo. 

```
function lint(options, files = [], fix = false) {
    const tslintConfigPath = path.join(options.gtsRootDir, 'tslint.json');
    const program = createProgram(options);
    const configuration = tslint_1.Configuration.findConfiguration(tslintConfigPath, '').results;
    const linter = new tslint_1.Linter({ fix, formatter: 'codeFrame' }, program);
    const srcFiles = files.length > 0 ? files : tslint_1.Linter.getFileNames(program);
    srcFiles.forEach(file => {
        const sourceFile = program.getSourceFile(file);
        if (sourceFile) {
            const fileContents = sourceFile.getFullText();
            linter.lint(file, fileContents, configuration);
        }
    });
    const result = linter.getResult();
    if (result.errorCount || result.warningCount) {
        options.logger.log(result.output);
        return false;
    }
    return true;
}
```

So perhaps this issue is only blocked by a release to npm.

**Comment by ofrobots** [comment]
Yes, I expect this should be resovled with our next release. Working on that now.

**Comment by ofrobots** [comment]
This was fixed by https://github.com/google/ts-style/pull/159.

**Comment by ofrobots** [comment]
Fixed in v0.7.0. Please try it out and report back if this is working better.
